src/RequestCookieProviderInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Cookies;
+
+use Yiisoft\Cookies\Exception\RequestCookieCollectionNotSetException;
+
+/**
+ * Provides a way to set the current cookie collection and then get it when needed.
+ */
+interface RequestCookieProviderInterface
```

Comment:
Add definition to package configurataion.

Reply:
done